tests/transformers/models/test_models.py

```diff
@@ -0,0 +1,93 @@
+# -----------------------------------------------------------------------------
+#
```

Comment:
This test infra will work for Causal LM models only.
Better to rename the file to reflect that